File: src/main/java/com/ajax_systems/akka/service/dto/CsvDTO.java
package com.ajax_systems.akka.service.dto;

import java.io.Serializable;


/**
 * A DTO for the Csv entity.
 */
public class CsvDTO implements Serializable {

    private String writer;

    private String value;

    private int row;

    public String getWriter() {
        return writer;
    }

    public void setWriter(String writer) {
        this.writer = writer;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }

    public int getRow() {
        return row;
    }

    public void setRow(int row) {
        this.row = row;
    }

    @Override
    public String toString() {
        return "CsvDTO{" +
                "writer='" + writer + '\'' +
                ", value='" + value + '\'' +
                ", row=" + row +
                '}';
    }
}

File: src/main/scala/com/ajax_systems/akka/service/dto/CsvDTO.scala
package com.ajax_systems.akka.service.dto


@SerialVersionUID(1L)
case class CsvDTO(var writer: String, var value: String, var row: Int) extends Serializable {
  def this() = this(null, null, 0)

  override def toString = "CsvDTO{" +
                  "writer='" + writer + '\'' +
                  ", value='" + value + '\'' +
                  ", row=" + row +
                  '}'
}